### PR TITLE
trillian: init at 1.3.11

### PR DESCRIPTION
**Pull request by adisbladis**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [x] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

#### Acknowledgement

This work is done within the context of [Trustix](https://nlnet.nl/project/Trustix/) and has been sponsored by a [NLNet Foundation NGI-0 PET](https://nlnet.nl/thema/NGIZeroPET.html) grant.

### CODE REVIEW

File: pkgs/tools/misc/trillian/default.nix [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    "cmd/createtree"
+    "cmd/deletetree"
+    "cmd/get_tree_public_key"
+    "cmd/updatetree"
+  ];
+
+  meta = {
+    homepage = "https://github.com/google/trillian";
+    description = "A transparent, highly scalable and cryptographically verifiable data store.";
+    license = [ lib.licenses.asl20 ];
```
Comment (SuperSandro2000) [inline pkgs/tools/misc/trillian/default.nix:40]: a with lib for meta wouldn't hurt.

File: pkgs/tools/misc/trillian/default.nix [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    "cmd/trillian_map_server"
+    "cmd/createtree"
+    "cmd/deletetree"
+    "cmd/get_tree_public_key"
+    "cmd/updatetree"
+  ];
+
+  meta = {
+    homepage = "https://github.com/google/trillian";
+    description = "A transparent, highly scalable and cryptographically verifiable data store.";
```
Comment (SuperSandro2000) [inline pkgs/tools/misc/trillian/default.nix:39]: > Transparent, highly scalable and cryptographically verifiable data store

File: pkgs/tools/misc/trillian/default.nix [hunk 3]
```diff
@@ -0,0 +1,43 @@
+{ stdenv
+, lib
+, buildGoModule
+, fetchFromGitHub
+}:
+let
```
[the remark this answers is not in the document]
Comment (adisbladis) [reply pkgs/tools/misc/trillian/default.nix:6]: No, `rec` is an anti-pattern: https://nix.dev/anti-patterns/index.html.

File: pkgs/tools/misc/trillian/default.nix (hunk 1 above)
Comment (adisbladis) [reply pkgs/tools/misc/trillian/default.nix:40]: No, `with` is an anti-pattern: https://nix.dev/anti-patterns/index.html.

**Pull request merged**